_posts/2017-11-03-2017-crystal-survey-2017-results.md

```diff
@@ -0,0 +1,230 @@
+---
+title: "Crystal Survey 2017 Results"
+author: sdogruyol
+summary: ""
+thumbnail: ""
+---
+
+This year, the Crystal team launched the first official [State of Crystal survey](https://crystal-lang.org/2017/07/31/launching-the-2017-state-of-crystal-survey.html). We're amazed by the fantastic feedback from the community.
+
+We've received ***757*** responses, thanks a lot to everyone who took the time to respond.  
+
+We've tried our best to read through all of your comments and suggestions. Without further ado, let's take a look at the results.
+
+
+![%19.9 Google or other search engines, %19.4 HackerNews, %15.6 Reddit %10.9 Twitter](https://cdn.pbrd.co/images/GP30DmW.png
+  "%19.9 Google or other search engines, %19.4 HackerNews, %15.6 Reddit %10.9 Twitter")
+(hover for more info)
+
+We wanted to learn how did you discover Crystal, it's interesting to see that a lot our respondents find Crystal from search engines. HackerNews, Reddit and Twitter shows the power of social media.

```

Comment:
We wanted to learn how you discovered Crystal, and were surprised to find that a lot of you arrived at Crystal through search engines. On the other hand, we confirmed how important HackerNews and Reddit to divulge young and small projects such as Crystal. Every now and then, someone from the community posts something interesting about Crystal, the post makes it to HN and Reddit front pages and in the next few days we notice a wave of new users and contributors on the different community channels.